pages/api/v1/dates/registration-closed.ts

```diff
@@ -0,0 +1,18 @@
+import { NextApiHandler } from 'next';
+import { protect } from '../../../../server/protect';
+import { getDate, postDate } from '../../../../server/dates';
+
+const regClosedHandler: NextApiHandler = async (req, res) => {
+  switch (req.method) {
+    case 'GET':
+      await getDate(req, res, 'registration-closed');
```

Comment:
with the above change, you'd be able to change these to the enum members, so something like SingletonType.RegistrationClosed, and wouldn't need to worry about typing the string wrong (since you'd get type safety).